vllm/v1/kv_cache_interface.py

```diff
@@ -71,6 +84,12 @@ def page_size_bytes(self) -> int:
 
 @dataclass
 class FullAttentionSpec(AttentionSpec):
+    # Some layers may be regarded as full attention layers in KV cache manager (
+    # blocks are allocated for all tokens), while computed as sliding window
+    # attention in model runner. In this case, we use FullAttentionSpec and
+    # record the sliding window size. Default to None for not using sliding
+    # window attention.
+    sliding_window: Optional[int] = None
```

Comment:
Is this for the case where the hybrid allocator is disabled? If so, please leave a comment.

Reply:
Yeah. I've updated the comment.